### DISCUSSION TITLE
Que faire quand le SIRET de plusieurs établissements est le même?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1154] [Question / réponse]
Plusieurs cas rencontrés récemment dans lesquels plusieurs établissements avaient le même SIRET :
- une école maternelle et une école élémentaire (l'école maternelle Épinette et l'école élémentaire Épinette à Libourne car elles font partie du même ensemble scolaire)
- un centre communal d'action social (le CCAS de Libourne) et une résidence de personnes âgées (Rés. Le Carmel à Libourne)
- un restaurant municipal et un restaurant du centre technique municipal (toujours à Libourne, ils ont tous deux le SIRET de la mairie)
- un ALSH et une école (car il s'agit du même lieu physique : l'école Simone Veil à Libourne devient un centre de loisirs les mercredi et pendant les vacances scolaires)

=> Laisse-t-on les SIRET tels quels pour ces particuliers?
=> Question de fond : le SIRET est-il la meilleure donnée de référencement des cantines?

**Comment by alemangui** [comment]
Excellente question, surtout qu'on en a parlé aujourd'hui lors du planning @valeriemerle @JenniferStephan 

Qu'est-ce qui prévoit la loi EGAlim ? Peut-on avoir plusieurs télédéclarations avec le même SIRET ? Comment la loi envisage les cas mentionnés par JB ?

**Comment by valeriemerle** [comment]
Coté GT suivi on envisage de demander le SIRET au CC (il est obligatoire pour l'agrément) et dans les satellites si possible. Si possible car ils n'ont pas tous de SIRET car ils n'ont pas fait les démarches tout simplement! par ailleurs dans le cas rapporté par JB il est possible que le siret donné soit celui du commanditaire (la commune ) et que ce n'est pas celui du site de distribution des repas parce que le gestionnaire ne le connait pas ou n'a pas pu le rentrer. Je jette un oeil au cas particulier de JB

**Comment by valeriemerle** [comment]
A mon avis il faut distinguer le commanditaire, la commune de Libourne, des etablissements où ont lieu les repas qui n'ont pas les mêmes SIRET et peut etre sont-ils reliés à une même cuisine centrale et certains sites sont alors des satellites. Il faudrait obtenir plus de renseignements sur le fonctionnement de ces différentes entités et leurs relations entre elles en matière de restauration. Dans l'exemple de JB, une rapide recherche sur internet indique que certains sites ont leur propres SIRET:
Commune de libourne : 21330243300015
les  ecoles maternelles et élémentaires dont en général deux niveaux regroupés dans un groupe scolaire appelé "ecole primaire" : l'école primaire d'Épinette , donc un seul SIRET : 21330243300148
un centre communal d'action social (le CCAS de Libourne) et une résidence de personnes âgées (Rés. Le Carmel à Libourne) :SIRET 26330240800046
un ALSH et une école (car il s'agit du même lieu physique : l'école Simone Veil à Libourne devient un centre de loisirs les mercredi et pendant les vacances scolaires) : dans ce cas, même SIRET mais double secteur d'activité? scolaire et loisirs?

**Reply by ghost** [reply]
- écoles maternelles et élémentaires : merci de la précision, j'ignorais.
- CCAS et résidence de personnes âgées : pourquoi la résidence Le Carmel a-t-elle le même SIRET que le CCAS de Libourne alors que d'autres résidences ont leur propre SIRET?
- ALSH et école : c'est bien ça, je vais donc ajouter le secteur "Loisirs" sur la fiche de la cantine

**Comment by valeriemerle** [comment]
par principe, il faudrait s'assurer que chaque cantine est son propre SIRET et qu'il ne soit pas possible de créer deux cantines avec le m^me SIRET (cas du lycée T Pesquet de Normandie créé par G Lebon en nov et re créé par le chef cuisinier le 25/03)
